src/SimpleVersion.Core/Pipeline/Formatting/Semver1FormatProcess.cs

```diff
@@ -9,6 +9,27 @@ public void Apply(VersionContext context)
         {
             var labelParts = new List<string>(context.Configuration.Label);
 
+            // Order of operations count here. Each part of the 'label' is added in the oder in which it is processed.
+
+            if (labelParts.Count == 0)
```

Comment:
Adding the branch name by default will need to go behind a configuration flag.  Or disabling it should.  I want there to be clear guidance/decisions and understanding on how versions are produced so it doesn't require in-depth knowledge of each step in the process to figure out why a specific version/label was produced.